gradle/distributionTesting.gradle

```diff
@@ -46,11 +46,12 @@ distributionTestTasks.all { DistributionTest task ->
 
     systemProperties['org.gradle.integtest.multiversion'] = project.hasProperty("testAllVersions") && project.testAllVersions ? 'all' : 'default'
 
-    systemProperties['org.gradle.integtest.mirrors.mavencentral'] = isCiServer ? 'http://dev12.gradle.org:8081/artifactory/repo1' : ''
-    systemProperties['org.gradle.integtest.mirrors.jcenter'] = isCiServer ? 'http://dev12.gradle.org:8081/artifactory/jcenter' : ''
-    systemProperties['org.gradle.integtest.mirrors.typesafemaven'] = isCiServer ? 'http://dev12.gradle.org:8081/artifactory/typesafe-maven-releases' : ''
-    systemProperties['org.gradle.integtest.mirrors.typesafeivy'] = isCiServer ? 'http://dev12.gradle.org:8081/artifactory/typesafe-ivy-releases' : ''
-    systemProperties['org.gradle.integtest.mirrors.google'] = isCiServer ? 'http://dev12.gradle.org:8081/artifactory/google' : ''
+    def mirrorUrls = collectMirrorUrls()
+    systemProperties['org.gradle.integtest.mirrors.mavencentral'] = mirrorUrls['mavencentral'] ?: ''
```

Comment:
:x: there's a lot of duplication here, I would keep the names of the expected mirrors in a list and loop, something like:
```Groovy
def mirrors = ['mavencentral', 'jcenter', 'typesafemaven', 'typesafeivy', 'google']
mirrors.each { mirror ->
    systemProperties["org.gradle.integtest.mirrors.$mirror"] = mirrorUrls[mirror] ?: ''
}
```

Reply:
Fixed.